orchestra/static/dist/main.css

```diff
@@ -2320,25 +2320,91 @@ a.logo {
     .todo-list .new-todo__description__datetime {
       display: inline-block;
       padding-left: 10px; }
-.existing-todos {
-  padding-top: 5px; }
-  .existing-todos .list-by-status {
-    padding-top: 20px; }
-    .existing-todos .list-by-status h4 {
-      padding-bottom: 10px; }
-    .existing-todos .list-by-status .todo input[type=checkbox] {
-      transform: scale(1.1); }
-    .existing-todos .list-by-status .todo label {
+.list-by-status {
```

Comment:
Looking at the video, there's a weird amount of vertical padding above the skipped item list. Can you remove that?:)

Reply:
I added that to keep both lists at same level. But I am good with removing it too.